### ISSUE TITLE
[bug] network tab unavailable

### ISSUE THREAD

**Issue by bingoct** [lllllllillllllillll/DweebUI#87]
saw the image problem in this https://github.com/lllllllillllllillll/DweebUI/issues/85. So i try the dev release. 

## compose.yaml
```yaml
version: "3.9"
services:
  dweebui:
    container_name: dweebui
    image: docker.io/lllllllillllllillll/dweebui:v0.70-dev
    networks:
      - traefik
    environment:
      PORT: ${port}
      SECRET: ${secret}
      HTTPS: false
    volumes:
      - type: bind
        source: ${dockerPath}
        target: /var/run/docker.sock
      - type: volume
        source: dweebui
        target: /app/config
    labels:
      # using traefik to expose service
      # enable traefik
      - traefik.enable=true
networks:
  traefik:
    driver: bridge
    external: true
volumes:
  dweebui:
    external: true
    name: dweebui
```

## Logs
DB Connection: ✔️
Synced Models: ✔️
Listening on http://localhost:8000
file:///app/controllers/networks.js:11
        let network_name = containers[i].HostConfig.NetworkMode;
                                                    ^

TypeError: Cannot read properties of undefined (reading 'NetworkMode')
    at Networks (file:///app/controllers/networks.js:11:53)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v22.3.0
DB Connection: ✔️
Synced Models: ✔️
Listening on http://localhost:8000
file:///app/controllers/networks.js:11
        let network_name = containers[i].HostConfig.NetworkMode;
                                                    ^

TypeError: Cannot read properties of undefined (reading 'NetworkMode')
    at Networks (file:///app/controllers/networks.js:11:53)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v22.3.0

**Comment by bingoct** [comment]
suggest: catch error instead of exist. 

The experience is terrible when clicking somewhere to trigger bugs, then the whole app is restart.

**Comment by lllllllillllllillll** [comment]
Yeah, it looks like the same issue. I need to add a try block.

**Comment by lllllllillllllillll** [comment]
Should be fixed in v0.70-dev now.

Commit: fc963b662ddf55fb74e71809502cf02603510f55

**Comment by lllllllillllllillll** [comment]
> The experience is terrible when clicking somewhere to trigger bugs, then the whole app is restart.

I've added pm2 to the image to prevent the app crashing on error: 6c15c28c2e3bb7a10470baeff2c235e1d9fbdb47
